dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -6259,117 +6206,6 @@
       "level_keys": [
         "AI Chat Test Level"
       ]
-    },
-    {
-      "chapter": 175,
-      "position": 2,
-      "activity_section_position": 2,
-      "assessment": false,
-      "properties": {
-      },
-      "bonus": false,
-      "seeding_key": {
-        "script_level.level_keys": [
-          "AI Chat Test Level - 2"
```

Comment:
Are you intending to remove these levels from the allthethings script?

Reply:
Oh, not sure what happened there, but should be fixed now.  I just wanted to add the one extra level.